projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/secrets/service/vault/VaultJobSecretsService.java

```diff
@@ -27,7 +27,7 @@
 @ConditionalOnProperty(value = "featureflag.vault.integration.enabled")
 public class VaultJobSecretsService implements com.vmware.taurus.secrets.service.JobSecretsService {
 
-  private static final int VAULT_SIZE_LIMIT_DEFAULT = 1048576; // 1 MB
+  public static final int VAULT_SIZE_LIMIT_DEFAULT = 1048576; // 1 MB
```

Comment:
Let's make it package private and not public. 

And add a comment saying "package private so it can be used in tests" to indicate why so.

Reply:
fixed.